system var: Active PD follower for handling region request (#15974)
pingcap/docs: system-variables.md

1 row in set (0.00 sec)
```

### pd_enable_follower_handle_region <span class="version-mark">New in v7.6.0</span>

> **Warning:**
>

> > The [Active PD Follower](/tune-region-performance.md#use-active-pd-follower-feature-to-enhance-the-scalability-of-pds-region-information-query-service) feature is experimental. It is not recommended that you use it in the production environment. This feature might be changed or removed without prior notice. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.

Reviewer: ```suggestion
> The [Active PD Follower](/tune-region-performance.md#use-the-active-pd-follower-feature-to-enhance-the-scalability-of-pds-region-information-query-service) feature is experimental. It is not recommended that you use it in the production environment. This feature might be changed or removed without prior notice. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.
```

> > The [Active PD Follower](/tune-region-performance.md#use-the-active-pd-follower-feature-to-enhance-the-scalability-of-pds-region-information-query-service) feature is experimental. It is not recommended that you use it in the production environment. This feature might be changed or removed without prior notice. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.

Reviewer: ```suggestion
> The [Active PD Follower](https://docs.pingcap.com/tidb/dev/tune-region-performance#use-the-active-pd-follower-feature-to-enhance-the-scalability-of-pds-region-information-query-service) feature is experimental. It is not recommended that you use it in the production environment. This feature might be changed or removed without prior notice. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.
```

> 

Reviewer 2: ```suggestion

> **Note:**
>
> This TiDB variable is not applicable to TiDB Cloud.

```

Reviewer 2: @ CabinfeverB, please check if the variable is applicable to TiDB Cloud. If not, please add the above note.

Reviewer: @qiancai pd_enable_follower_handle_region is also applicable to TiDB Cloud.

- Scope: GLOBAL
- Persists to cluster: Yes
- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): No
- Type: Boolean
- Default value: `OFF`
- This variable controls whether to enable the Active PD Follower feature (currently only applicable for requests related to Region information). When the value is `OFF`, TiDB only obtains Region information from the PD leader. When the value is `ON`, TiDB evenly distributes requests related to Region information to all PD servers. PD followers can also directly handle Region requests, thereby reducing the CPU pressure on the PD leader.
- Scenarios for enabling Active PD Follower:
    * In a cluster with a large number of Regions, the PD leader experiences high CPU load due to the overhead of handling heartbeats and scheduling tasks, leading to CPU resource exhaustion.
    * The TiDB cluster has many TiDB instances, and there is a high concurrency of requests for Region information, leading to high CPU pressure on the PD leader.

> 

Author: 因为之前的格式我是抄的这个的，然后 aolin 提了 comment，我就说一起改一下, 不知道这样规不规范

> 

Reviewer: Done https://github.com/pingcap/docs/pull/15974/commits/019d803d16911ba6ad51e3ad154f86b8636fa4e4
